src/detector.rs

```diff
@@ -896,12 +905,28 @@ impl LanguageDetector {
         if total_language_counts.len() == 2
             && cfg!(feature = "chinese")
             && cfg!(feature = "japanese")
-            && total_language_counts.contains_key(&Some(Language::from_str("Chinese").unwrap()))
-            && total_language_counts.contains_key(&Some(Language::from_str("Japanese").unwrap()))
+            && total_language_counts.contains_key(&Some(Language::Chinese))
```

Comment:
Please replace `Language::Chinese` with `Language::from_str("Chinese")` as it was before. The same goes for Japanese. Otherwise, the code won't compile if Chinese and / or Japanese are not among the selected language dependencies.